etc/spack/defaults/packages.yaml

```diff
@@ -25,9 +25,13 @@ packages:
       fftw-api: [fftw, amdfftw]
       flame: [libflame, amdlibflame]
       fuse: [libfuse]
-      gl: [mesa+opengl, mesa18, opengl]
       glu: [mesa-glu, openglu]
-      glx: [mesa+glx, mesa18+glx, opengl]
```

Comment:
Let's leave `mesa18` in here for now.  It can be deprecated and removed in a separate pr.